package.json

```diff
@@ -1,73 +1,48 @@
 {
   "name": "ngx-swiper-wrapper",
-  "description": "Angular wrapper library for Swiper",
-  "bugs": "https://github.com/zefoy/ngx-swiper-wrapper/issues",
-  "license": "MIT",
   "version": "4.6.6",
-  "main": "bundles/ngx-swiper-wrapper.umd.js",
-  "module": "./dist/index.js",
-  "typings": "./dist/index.d.ts",
-  "tslint": {
-    "extends": "./tslint.json"
-  },
-  "stylelint": {
-    "extends": "./stylelint.json"
-  },
+  "private": true,
+  "license": "MIT",
   "scripts": {
     "init": "npm install",
     "start": "npm run build",
     "watch": "watch 'npm run build' src/",
     "clean": "rm -rf dist bundles node_modules",
+    "link": "ngm link -p src",
     "lint": "npm run lint:js && npm run lint:scss",
-    "build": "npm run build:umd && npm run build:aot",
+    "build": "ngm build -p src --clean --skip-bundles",
     "deploy": "npm run build && npm run inline && npm publish",
     "inline": "ng2-inline --compress -b dist/lib/ -o ./ 'dist/lib/*.js'",
     "lint:ts": "tslint 'src/**/*.ts'",
-    "lint:scss": "stylelint 'src/**/*.scss' --syntax scss",
-    "build:js": "ngc -p src/tsconfig.json",
-    "build:css": "node-sass src/ -o dist/ --source-map=true",
-    "build:html": "cpx src/lib/*.html dist/lib/",
-    "build:umd": "webpack --config config/webpack.config.js",
-    "build:aot": "npm run build:js && npm run build:css && npm run build:html"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/zefoy/ngx-swiper-wrapper.git"
+    "lint:scss": "stylelint 'src/**/*.scss' --syntax scss"
   },
   "dependencies": {
     "swiper": "^3.0.0"
   },
   "devDependencies": {
-    "@angular/common": "^4.0.0",
-    "@angular/compiler": "^4.0.0",
-    "@angular/compiler-cli": "^4.0.0",
-    "@angular/core": "^4.0.0",
-    "@angular/platform-browser": "^4.0.0",
+    "@angular/common": "4.4.3",
+    "@angular/compiler": "4.4.3",
+    "@angular/compiler-cli": "4.4.3",
+    "@angular/core": "4.4.3",
+    "@angular/platform-browser": "4.4.3",
     "@types/node": "~6.0.0",
     "@types/swiper": "~2.0.0",
     "angular2-inline-template-style": "~1.0.0",
-    "angular2-template-loader": "~0.6.0",
-    "awesome-typescript-loader": "~3.2.0",
-    "codelyzer": "~3.1.0",
-    "core-js": "~2.4.0",
-    "cpx": "~1.5.0",
-    "node-sass": "~4.5.0",
-    "raw-loader": "~0.5.0",
-    "rxjs": "~5.4.0",
-    "sass-loader": "~6.0.0",
-    "string-replace-loader": "~1.1.0",
-    "strip-sourcemap-loader": "~0.0.0",
+    "codelyzer": "3.2.1",
+    "core-js": "2.4.1",
+    "ngm-cli": "^0.8.3",
```

Comment:
strict ?

Reply:
Version has been taken from package.json of other sub-projects